bin/zold

```diff
@@ -125,6 +127,10 @@ Available options:"
     exit
   end
 
+  version_file = Zold::VersionFile.new('.zoldata')
+  updates = Zold::Upgrades.new(version_file)
+  updates.run
```

Comment:
@botanicus this looks a bit messy to me. First, try to stay away from redundant variables: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html Second, why a "file" encapsulates a directory? Doesn't read right to me. How about this:

```ruby
Zold::Upgrades.new(Zold::Version.new('.zoldata/version')).run
```